### DISCUSSION TITLE
Product name in second language

### DISCUSSION THREAD

**Discussion by magedfathi** [Blair2004/NexoPOS#413] [Ideas]
here in The Arabian Gulf it is mandatory to give the receipt in Arabic and English language so i would like to suggest to have not a mandatory field for product name in arabic or any second language
product name English
product name Arabic
so while print the receipt it will show name in english - name in arabic - price

the following photo i am writing the name in arabic and english in one filed but it is looking good in the POS

https://user-images.githubusercontent.com/87772530/147046273-154920c7-87ef-440a-b2d9-e45d42b104d6.PNG

**Comment by Blair2004** [comment]
Hi, 
Thank you we'll schedule this for NexoPOS 4.7.x
Thanks for your contribution.

**Comment by magedfathi** [comment]
i would like to remind you for your promise and the following screen is shown how it looks

![issue8](https://user-images.githubusercontent.com/87772530/186639894-b780bc9d-efec-4e71-9487-f646eba7c2cd.jpg)
